charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -1,6 +1,48 @@
 {
   "title": "datacatalog.json",
   "definitions": {
+    "CreateAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID",
+        "resourceMetadata",
+        "resourceDetails"
+      ],
+      "properties": {
+        "credentials": {
+          "description": "This optional field has the vault plugin path where the destination data credentials will be stored as kubernetes secrets",
+          "type": "string"
+        },
+        "destinationAssetID": {
+          "description": "This is an optional field provided to give information about the asset id to be used for the created asset.",
+          "type": "string"
+        },
+        "destinationCatalogID": {
+          "description": "This has the information about the destination catalog id that new asset that will be created with the information provided in ResourceMetadata and Details field will be stored.",
```

Comment:
ID of the catalog where the new asset will be registered.

Reply:
Fixed. Please check. Thanks.